e2e/kubernetes/resource_kubernetes.go

```diff
@@ -0,0 +1,666 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/e2eterraformprovider/terraform-provider-e2e/client"
+	"github.com/e2eterraformprovider/terraform-provider-e2e/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceKubernetesService() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the Kubernetes service",
+				ForceNew:    true,
+			},
+			"version": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Version of the Kubernetes service",
+				ForceNew:    true,
+			},
+			"project_id": {
+				Type:        schema.TypeInt,
+				Required:    true,
+				ForceNew:    true,
```

Comment:
Please do not use forcenew true for project id field we do not want user to change this field at all.

Reply:
Done